worlds/lingo/data/LL1.yaml

```diff
@@ -73,10 +75,6 @@
   #                   will be covered by a single item.
   # - include_reduce: Door checks are assumed to be EXCLUDED when reduce checks
   #                   is on. This option includes the check anyway.
-  # - junk_item:      If on, the item for this door will be considered a junk
-  #                   item instead of a progression item. Only use this for
-  #                   doors that could never gate progression regardless of
-  #                   options and state.
```

Comment:
I want to add settings-dependent item classification later. But that's probably something I'll handle on a case-by-case basis rather than have in the config file. `junk_item` isn't used by anything at this point, so I'm getting rid of it.